.env-sample

```diff
@@ -0,0 +1,5 @@
+DB_USERNAME='DB_USERNAME'
+DB_PASSWORD='DB_PASSWORD'
```

Comment:
now that you have the env files you should be removing your password etc from your model index and replacing it with the config sequelize